llvm/test/tools/sycl-post-link/sycl-external-funcs/split-global.ll

```diff
@@ -0,0 +1,22 @@
+; RUN: sycl-post-link -ir-output-only -split=auto -S %s -o %t.ll
+; RUN: FileCheck %s -input-file=%t.ll
+
+; This test checks that unreferenced functions with sycl-module-id
+; attribute are not dropped from the module after splitting
+; in global mode.
```

Comment:
What do you mean by `global` mode?

Reply:
I meant the mode when no splitting happens (corresponds to `Scope_Global`).